nixos/modules/security/acme/default.nix

```diff
@@ -285,10 +279,10 @@ let
 
     selfsignService = lockfileName: {
       description = "Generate self-signed certificate for ${cert}";
-      after = [ "acme-selfsigned-ca.service" "acme-fixperms.service" ] ++ lib.optional (cfg.maxConcurrentRenewals > 0) "acme-lockfiles.service";
-      requires = [ "acme-selfsigned-ca.service" "acme-fixperms.service" ] ++ lib.optional (cfg.maxConcurrentRenewals > 0) "acme-lockfiles.service";
+      after = [ "acme-setup.service" ];
+      requires = [ "acme-setup.service" ];
 
-      path = with pkgs; [ minica ];
+      path = [ pkgs.minica ];
```

Comment:
It's only used in one place, might as well just use `lib.getExe` there.

Reply:
It's used in two places, as you noted. I feel that declaring path entries is more semantically logical than using lib functions to achieve the same result.

Reply:
Ok this is just a nit, fine by me!